microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -3,254 +3,284 @@ description: The latest experimental features in Microsoft Edge DevTools
 title: Experimental features
 author: MSEdgeTeam
 ms.author: msedgedevrel
-ms.date: 03/15/2021 
+ms.date: 08/31/2021
 ms.topic: article
 ms.prod: microsoft-edge
 keywords: microsoft edge, web development, f12 tools, devtools, experiment
 no-loc: ["Enable webhint", "Enable Network Console", "Source Order Viewer", "Enable Composited Layers in 3D View", "Enable new Font Editor tool within the Styles pane", "Enable new CSS Flexbox debugging features", "Enable + button tab menus to open more tools", "Enable Welcome tab", "3D View", "Turn on support to move tabs between panels", "Match keyboard shortcuts in the DevTools to Microsoft Visual Studio Code", "Edit keyboard shortcuts for any action in the DevTools", "Turn on new CSS grid debugging features", "Emulation: Support dual screen mode"]
 ---
-# Experimental features  
+# Experimental features
 
-Microsoft Edge DevTools provide access to experimental features that are still in development.  You may test and [provide feedback](#providing-feedback-on-experimental-features) before each feature is released.  
+Microsoft Edge DevTools provide access to experimental features that are still in development.  You can test and [provide feedback](#providing-feedback-on-experimental-features) before each feature is released.
 
-While experimental features are available in all channels of Microsoft Edge, you may get the latest experimental features using the Microsoft Edge Canary channel.  
+Experimental features are available in all of the preview channels of Microsoft Edge.  You can get the latest experimental features by using the Microsoft Edge Canary channel.  This article only covers selected experimental features.  For the full list, see the **Settings** > **Experiments** page in DevTools.
```

Comment:
In fact, experimental features are available everywhere, not just on preview channels. If you open the experiment section of devtools in release now (92), you will see that experiments are there too, some of them enabled by defaults, some not.

I would almost delete the "Experimental features are available in all of the preview channels of Microsoft Edge" sentence and start the paragraph with "You can get the latest experimental features by using the Microsoft Edge Canary channel, but all other channels of Microsoft Edge also have experimental features that can be accessed." 
And then "This article only covers ..." (no change).

What do you think?